File: beecrowd/problems/iniciante/js/1002.js
const input = require("fs").readFileSync("./dev/stdin", "utf8");

const [raio] = input.split("\n").map(item => parseFloat(item));

const N = 3.14159;

const area = N * Math.pow(raio, 2);

console.log(`A=${area.toFixed(4)}`);

File: beecrowd/problems/iniciante/java/1002.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

public class Main {

    public static void main(String[] args) throws IOException {
 
        Scanner read = new Scanner(System.in);
        
        double raio = read.nextDouble();
        double N = 3.14159;
        double area;
        
        area = N * Math.pow(raio, 2);
        
        System.out.println("A=%.4lf" + area);

    }

}